src/test/java/Test1/Test1.java

```diff
@@ -0,0 +1,6 @@
+package Test1;
+
+public class Test1 {
+
+    //test1
```

Comment:
u should run test2 here

Reply:
updated in [9399400](https://github.com/euroTech-sdet/Batch4_GitPractice/pull/5/commits/9399400f6a2a605de460de8e225ec0d6de0f3a51)